integration-tests/utils/zendesk/approveZendeskTicket.ts

```diff
@@ -0,0 +1,25 @@
+import axios from 'axios'
+import { authoriseAs } from './authoriseAs'
+import { ticketApprovalData } from '../../constants/requestData'
+import {
+  ZENDESK_AGENT_EMAIL,
+  ZENDESK_BASE_URL,
+  ZENDESK_TICKETS_ENDPOINT
+} from '../../constants/zendeskParameters'
+
+export const approveZendeskTicket = async (ticketId: string) => {
+  try {
+    await axios({
+      url: `${ZENDESK_BASE_URL}${ZENDESK_TICKETS_ENDPOINT}/${ticketId}`,
+      method: 'PUT',
+      headers: {
+        Authorization: authoriseAs(ZENDESK_AGENT_EMAIL),
+        'Content-Type': 'application/json'
+      },
+      data: ticketApprovalData
```

Comment:
Can we have the assertions back in a `then`? Especially to check the new ticket status.

Reply:
I removed the assertions on purpose, as I didn't see the need to assert that Zendesk response was a 201, etc. as I don't think we need to be testing the Zendesk API.

A successful response means it has done its job surely?

Reply:
Actually on further reflection, it makes sense to check the ticket contains the approved tag as this is a custom thing we configure in zendesk, so have added this back.